src/main/webapp/assets/modules/contest-detail/challengeCriteriaInputDirective.js

```diff
@@ -0,0 +1,15 @@
+techlooper.directive('challengeCriteriaInput', function () {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/contest-detail/challengeCriteriaInput.html",
+    scope: {
+      cri: "=",
+      contestDetail: "="
+    },
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.